backend/src/main/java/io/debezium/configserver/rest/ConnectorURIs.java

```diff
@@ -0,0 +1,18 @@
+package io.debezium.configserver.rest;
+
+public interface ConnectorURIs {
```

Comment:
Let's make this a final class rather; [see here](https://rules.sonarsource.com/java/RSPEC-1214) why implementing constant interfaces is discouraged.

Reply:
oh yes yes yes yes! very good